mygame.py

```diff
@@ -67,6 +70,16 @@ def Step(self):
             elif lowcmd in self.npc_commands:
                 for greeting in self.worldmap.greetnpcs(self.player1.position.x, self.player1.position.y):
                     self.graphics.RenderText(greeting)
+            elif lowcmd in self.item_commands:
+                item_get = input("What will you pick up?")
+                item = self.worldmap.pickupitems(self.player1.position.x, self.player1.position.y, item_get)
+                if item == "You cannot get that.":
+                    print("You cannot get that.")
+                else:
+                    self.player1.additemtoinv(item)
+            elif lowcmd in self.inventory_commands:
```

Comment:
I see what you mean here, maybe you can at least say something like 
RenderText("You are carrying:")
for ...
   RenderText("    " + item)

And we can brainstorm a better graphics engine.

Reply:
The issue I think is that it shows you everything each passthrough.  Whereas if I type inventory I would just want to see that, or if I type look I would just want to see the results.